docs/core/resilience/snippets/nuget.config

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!-- Define the package sources, nuget.org and contoso.com. -->
```

Comment:
Comments look a bit copy-pasta as there's nothing contoso in this file?

Reply:
This file will be deleted, and it's not shown to the reader. It was for our build system to ensure that we could get the packages to compile the source.